com.unity.render-pipelines.universal/Runtime/Passes/CopyDepthPass.cs

```diff
@@ -52,9 +52,9 @@ public override void OnCameraSetup(CommandBuffer cmd, ref RenderingData renderin
             descriptor.graphicsFormat = isDepth ? GraphicsFormat.D32_SFloat_S8_UInt : GraphicsFormat.R32_SFloat;
             descriptor.msaaSamples = 1;
 #if UNITY_EDITOR
-            ConfigureTarget(destination, destination, GraphicsFormat.R32_SFloat, descriptor.width, descriptor.height, descriptor.msaaSamples);
+            ConfigureTarget(destination, destination);
```

Comment:
Would need to revisit this. Again, unless doing backwards compatibility, destination should be an RTHandle of color XOR depth

Reply:
yeah, this is basically a hack to make dx11 work properly. In this case - depth is actually not used, but if we don't specify it - it will render to backbuffer and mess up all of the depth in further passes

Reply:
Could you add a comment to that line saying this?

Reply:
sure, will do

Reply:
Done in eb5919e